pkg/controller/manager/manager_controller_test.go

```diff
@@ -78,7 +78,29 @@ var _ = Describe("Manager controller tests", func() {
 		}
 		err := c.Create(ctx, instance)
 		Expect(err).NotTo(HaveOccurred())
-		instance, err = GetManager(ctx, c)
+		instance, err = GetManager(ctx, c, "")
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	It("should create and query multiple tenant manager instances", func() {
```

Comment:
I think we should call out that this test is using a mock client rather than a real one - the operator code that is being executed here is non-zero, but it's a pretty small test (it's just verifying the small bit of code wrapped around the client in GetManager).

It would also be good to have a negative test case - e.g., call `GetManager` for a namespace that doesn't have a manager instance.